backend/sample/schema.json

```diff
@@ -3,33 +3,22 @@
         {
             "id" : "__header__",
             "value" : [
-                {
-                    "id" : "stopwords:IDX",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                },
-                {
-                    "id" : "synonyms:IDX",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                }
+                {"type" : "string"}
             ]
         },
         {
-            "id" : "__comments__",
-            "value" : [
-                {
-                    "id" : "comment",
-                    "value" : {"type" : "string"},
-                    "block_index" : {"type" : "integer"},
-                    "block_subindex" : {"type" : "integer"}, 
-                    "preceding_lines" : {"type" : "integer"}
-                }
-            ]
+            "id" : "stopwords:IDX",
+            "value" : {"type" : "string"},

```

Comment:
> It's a list of tag and tagid. Something like `[{"tag": "en:The", "tagid": "en:the"}, {"tag: "en:Some", "tagid": "en:some"}]`.

I think this would be a better representation, since each tag and tagid would be in one dictionary and easy to query.

Reply:
For stopwords we only have one language per entry so maybe something like:
```
...
"lang": "en", 
"tag": ["The", "Some"], 
"tagid": ["the", "some"]
...
```
is the right way.